ui/src/components/DownloadButton/DownloadButton.stories.ts

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import DownloadButton from './DownloadButton';
+
+const meta = {
+    component: DownloadButton,
+    title: 'Components/DownloadButton',
+} satisfies Meta<typeof DownloadButton>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Base: Story = {
+    args: {
+        // using index.json as it needs to be from the same domain
+        fileUrl: 'http://localhost:6006/index.json',
```

Comment:
can't it be relative? It won't be localhost when it will be deployed [here](https://splunk.github.io/addonfactory-ucc-generator/storybook)

Reply:
yea, relative works too, forgot about this page